packages/web/src/pages/dev-tools/messages.ts

```diff
@@ -0,0 +1,38 @@
+export const messages = {
+  pageTitle: 'Developer Tools',
+  pageDescription:
+    'This page provides utilities for developers to test and debug the application.',
+  featureFlagsTitle: 'Feature Flags',
+  featureFlagsDescription:
+    'Override feature flags for testing purposes. Changes require a page refresh to take effect.',
+  featureFlagsButton: 'Open Feature Flag Editor',
+  discoveryNodeTitle: 'Discovery Node Selector',
+  discoveryNodeDescription:
+    "Select a specific Discovery Node to connect to. Alternatively, press 'D' key to toggle this tool.",
+  discoveryNodeButton: 'Open Discovery Node Selector',
+  confirmerPreviewTitle: 'Confirmer Preview',
+  confirmerPreviewDescription:
+    "Preview the state of the confirmer, which manages transaction retries. Alternatively, press 'C' key to toggle this tool.",
+  confirmerPreviewButton: 'Open Confirmer Preview',
+  signatureDecoderTitle: 'Signature Decoder',
+  signatureDecoderDescription:
+    'Input a raw Secp256k1 instruction hex string (e.g., the "Instruction Data" from Solana Explorer for a Secp256k1 SigVerify Precompile instruction) to decode the underlying Claimable Tokens message.',
```

Comment:
nit: technically this is only the instruction data, not the full instruction. An instruction is made up of three parts: programId, accounts, and data.